test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs

```diff
@@ -957,6 +957,16 @@ private class SDKTypeBindings_BlobStorage
             }
 
 
+            [Function("BlobClientToBlobStringFunction")]
+            [BlobOutput("container1/hello.txt", Connection = "MyOtherConnection")]
+            public object BlobClientToBlobStringFunction(
+                [BlobTrigger("container2/%file%")] BlobClient blob,
+                [BlobInput("container2/%file%")] string blobinput)
+            {
+                throw new NotImplementedException();
+            }
+
+
             [Function("BlobClientToBlobStringFunction")]
             [BlobOutput("container1/hello.txt", Connection = "MyOtherConnection")]
```

Comment:
Add tests for Table extension also similar to Blobs which validate if SupportsDeferredBinding is being assigned correctly by `FunctionMetadataGenerator` based on `binding type` for both the cases - types that support and types that bypass deferred binding

Reply:
Added tests for both files!